[13.0][MIG] hr_employee_ppe: Migration to 13.0

marcelsavegnago opened the pull request:
cc @eduaparicio

eduaparicio committed:
[12.0][ADD] hr_employee_ppe

eduaparicio committed:
[12.0][UPD] Changes proposed by the Community for the first version implemented.

eduaparicio committed:
[12.0][UPD] Location field removed. Indications field included. Minor layout improvements and lynt corrections.

oca-travis committed:
[UPD] Update hr_employee_ppe.pot

OCA-git-bot committed:
[UPD] README.rst

marcelsavegnago committed:
[IMP] hr_employee_ppe: Fix Maintainers and module credits

OCA-git-bot committed:
[UPD] README.rst

marcelsavegnago committed:
[IMP] Tests and fixes

lfreeke commented:
I don't want to hold up this pull request, but I wonder if we cannot make this module more general, so it can be used for other assets too? 

What do you all think about that?

marcelsavegnago commented:
> I don't want to hold up this pull request, but I wonder if we cannot make this module more general, so it can be used for other assets too?
> 
> What do you all think about that?

No problem it is not delaying it, I am still waiting for the PR https://github.com/OCA/hr/pull/975 to be accepted so that I can redo this branch with the corrections made in 12.0

I considered that the hr_maintenance module would be the closest module to this type of control and that is why we suggest a specific module for the management of PPE. But I am totally open to suggestions.

https://github.com/odoo/odoo/blob/b05a72d3cc181c601807b5e633766d504e06bb3a/addons/hr_maintenance/__manifest__.py#L10

oca-travis committed:
[UPD] Update hr_employee_ppe.pot

OCA-git-bot committed:
hr_employee_ppe 12.0.1.1.0

oca-travis committed:
[UPD] Update hr_employee_ppe.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_employee_ppe
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_ppe/

marcelsavegnago committed:
[IMP] hr_employee_ppe: black, isort, prettier

marcelsavegnago committed:
[MIG] hr_employee_ppe: Migration to 13.0

marcelsavegnago commented:
ping @OCA/human-resources-maintainers

rvalyi reviewed (approved):
tested on Runbot.

lfreeke commented:
> > I don't want to hold up this pull request, but I wonder if we cannot make this module more general, so it can be used for other assets too?
> > What do you all think about that?
> 
> No problem it is not delaying it, I am still waiting for the PR #975 to be accepted so that I can redo this branch with the corrections made in 12.0
> 
> I considered that the hr_maintenance module would be the closest module to this type of control and that is why we suggest a specific module for the management of PPE. But I am totally open to suggestions.
> 
> https://github.com/odoo/odoo/blob/b05a72d3cc181c601807b5e633766d504e06bb3a/addons/hr_maintenance/__manifest__.py#L10

Indeed this seems to be the closest module, but I would also agree that this wouldn't really fit with PPE. But I also want to prevent we have a module for everything :)

marcelsavegnago commented:
ping @OCA/human-resources-maintainers

marcelsavegnago commented:
ping @mileo @hendrixcosta

Saran440 commented on hr_employee_ppe/__manifest__.py:
```diff
@@ -0,0 +1,25 @@
+
+{
+    "name": "Personal Protective Equipment (PPE) Management",
+    "version": "13.0.1.1.0",
```
```suggestion
    "version": "13.0.1.0.0",
```

Saran440 commented on hr_employee_ppe/__manifest__.py:
```diff
@@ -0,0 +1,25 @@
+    "author": "Escodoo, Odoo Community Association (OCA)",
+    "maintainers": ["marcelsavegnago", "eduaparicio"],
+    "images": ["static/description/banner.png"],
+    "website": "https://github.com/oca/hr",
```
```suggestion
    "website": "https://github.com/OCA/hr",
```

Saran440 reviewed (commented):
Code reviewed.

Saran440 commented:
/ocabot merge nobump
